src/client/containers/Builds/index.js

```diff
@@ -0,0 +1,84 @@
+/* @flow */
+
+import React, { PureComponent } from 'react'
+import Col from 'react-bootstrap/lib/Col'
+import Row from 'react-bootstrap/lib/Row'
+import { getBuilds } from 'ducks/bfstats/selectors'
+
+import ListGroupItem from 'react-bootstrap/lib/ListGroupItem'
+import { connect } from 'react-redux'
+
+
+type BuildType = {
+  builder: {
+    name: string
+  },
+  result: number
+}
+
+type PropsType = {
+  repository: string,
+  revision: string,
+  builds: Array<BuildType>,
+}
+
+const subHeader = text => (
+  <div style={{ color: '#8c8c8c', fontSize: '13px' }}>
+    {text}
+  </div>
+)
+
+const success = {
+  borderLeft: '5px solid rgb(214, 247, 229)',
+}
+
+const headerColumnStyle = {
+  textTransform: 'uppercase',
+  color: '#10121b',
+}
+
+const approvedColor = '#3f855b'
+
+class Builds extends PureComponent {
+
+  props: PropsType
+
+  render() {
+    if (!this.props.builds || !this.props.builds.length) {
+      return null
+    }
+    return (
+      <ListGroupItem style={success}>
+        HAHAHA
```

Comment:
yes, I mentioned this in PR description, component UI will be finished in next PR